code/style.css

```diff
@@ -0,0 +1,64 @@
+* {
+    padding: 0;
+    margin: 0;
+    box-sizing: border-box;
+  }
+  
+  body {
+    background: #eee;
+    font-family: sans-serif;
+  }
+  
+  .banner-area {
+    width: 100%;
+    height: 50vh;
+    background-image: url(./pizza.jpg);
+    filter: opacity(0.9);
+    background-size: cover;
+  }
```

Comment:
Indentation (the elements should be on the same level - now all elements except `*` are indented). This goes for the whole CSS-file.

```
* {
  padding: 0;
  margin: 0;
  box-sizing: border-box;
}

body {
  background: #eee;
  font-family: sans-serif;
}

.banner-area {
  width: 100%;
  height: 50vh;
  background-image: url(./pizza.jpg);
  filter: opacity(0.9);
  background-size: cover;
}
```